crates/universal-sierra-compiler-api/src/lib.rs

```diff
@@ -15,10 +15,16 @@ use shared::command::CommandExt;
 
 mod command;
 
+/// The offset of the source sierra statement within the CASM bytecode
+type CasmCodeOffset = usize;
+
+/// The index of the sierra statement in the instructions vector.
+type InstructionIdx = usize;
+
 #[derive(Serialize, Deserialize)]
 pub struct AssembledProgramWithDebugInfo {
     pub assembled_cairo_program: AssembledCairoProgramWithSerde,
-    pub debug_info: Vec<(usize, usize)>,
+    pub debug_info: Vec<(CasmCodeOffset, InstructionIdx)>,
```

Comment:
Does debug info has a property of uniqueness on InstructionIdx?

Reply:
yeah probably, it should